modules/comments.php

```diff
@@ -3,12 +3,13 @@
 /**
  * Module Name: Comments
  * Module Description: Let readers use WordPress.com, Twitter, Facebook, or Google+ accounts to comment
+ * Jumpstart Description: Let readers use WordPress.com, Twitter, Facebook, or Google+ accounts to comment.
```

Comment:
Let's just change these to be "Google" accounts, since that's how they more broadly refer to them these days.